apps/test/unit/instructionsComponentsTest.js

```diff
@@ -10,73 +10,67 @@ describe('instructions components', () => {
 
   describe('MarkdownInstructions', function() {
     it('standard case had top padding and no left margin', function() {
-      var dom = mount(
-        <div>
-          <StatelessMarkdownInstructions
-            markdown="md"
-            markdownClassicMargins={false}
-            inTopPane={false}
-            noInstructionsWhenCollapsed={true}
-          />
-        </div>
+      const wrapper = mount(
+        <StatelessMarkdownInstructions
+          markdown="md"
+          markdownClassicMargins={false}
+          inTopPane={false}
+          noInstructionsWhenCollapsed={true}
+        />
```

Comment:
- Changing `dom` to `wrapper` to follow enzyme conventions.
- Removing the enclosing `<div>` element because it's not relevant to the test.